src/tests/system/tests/test_sssctl.py

```diff
@@ -67,3 +68,55 @@ def test_sssctl__check_invalid_id_provider(client: Client):
         "[rule/sssd_checks]: Attribute 'id_provider' in section 'domain/test' has an invalid value: invalid"
         in output.stdout_lines[1]
     )
+
+
+@pytest.mark.ticket(bz=1902280)
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_sssctl__reset_cached_timestamps(client: Client, ldap: LDAP):
+    """
+    :title: fix sss_cache to also reset cached timestamp
+    :setup:
+        1. Add user to SSSD
+        2. Add group to SSSD
+        3. Start SSSD
+    :steps:
+        1. Call getent group
+        2. Set domain config
+        3. Restart SSSD
+        4. Call getent group
+        5. Modify group entry in LDAP
+        6. Call 'sss_cache -E'
+        7. Call getent group
+    :expectedresults:
+        1. Group is properly cached, user is its member
+        2. Config set successfully
+        3. Restarted successfully, config is applied
+        4. Group is properly cached, user is its member
+        5. Member of group is deleted, group entry changed
+        6. Whole cache is invalidated
+        7. Deleted user is not member of group anymore
+    :customerscenario: True
+    """
+    u = ldap.user("user1").add()
+    ldap.group("group1").add().add_member(u)
+    client.sssd.start()
+
+    res1 = client.tools.getent.group("group1")
+    assert res1 is not None
+    assert "user1" in res1.members
+
+    client.sssd.domain["ldap_schema"] = "rfc2307bis"
+    client.sssd.domain["ldap_group_member"] = "uniquemember"
+    client.sssd.domain["debug_level"] = "9"
```

Comment:
Is there a specific reason why these domain configuration settings are updated in the middle of the test, instead of at the beginning? It is a bit confusing in my opinion.

Reply:
The specific reason why it's this way is because it works :smile:  The other way around it does not, tests fails on line 115 - user is not member of the group. It has something to do with this ldap_scheme.